checkers_bot_tournament/game.py

```diff
@@ -243,11 +250,11 @@ def make_move(self) -> Optional[Result]:
 
         return None
 
-    def _record_capture(self) -> None:
+    def _record_capture(self, captures: int) -> None:
         if self.current_turn == Colour.WHITE:
-            self.white_num_captures += 1
+            self.white_num_captures += captures
         else:
-            self.black_num_captures += 1
+            self.black_num_captures += captures
```

Comment:
I can't comment on line 270 for some reason, but can you remove the TODO comment on that line?